app/assets/stylesheets/components/_form.scss

```diff
@@ -0,0 +1,7 @@
+form {
```

Comment:
How about moving the `_form` and `_input` to the `base` folder instead of the component folder 🤔 

Also we could merge the content into the same file as the `input` can be in the `form` term.

For more information, you can check it on our Compass https://compass.nimblehq.co/development/code-conventions/css/#base

Reply:
Fixed